bandit/plugins/django_xss.py

```diff
@@ -87,7 +87,7 @@ def evaluate_var(xss_var, parent, until, ignore_nodes=None):
     if isinstance(xss_var, ast.Name):
         if isinstance(parent, ast.FunctionDef):
             for name in parent.args.args:
-                arg_name = name.id if six.PY2 else name.arg
+                arg_name = name.id
```

Comment:
This would be rewritten as such @lukehinds:

```py
if six.PY2:
    arg_name = name.id
else:
    arg_name = name.arg
```

Does that help to explain what I was trying to say earlier? We want `name.arg` for Py3

Reply:
ack, makes sense now